priv/rel/files/boot

```diff
@@ -273,9 +273,9 @@ case "$1" in
             echo "Unable to write to $PIPE_DIR. Quitting."
             exit 1
         fi
-
+        EXTRA_ARGS_LIST=$3
```

Comment:
I tested this with multiple arguments aka app --api --v2 and my application received all the arguments. But I can check again with the -- and see if the arguments get passed.